volatility3/framework/plugins/yarascan.py

```diff
@@ -56,15 +69,62 @@ def __call__(
                 for offset, name, value in match.strings:
                     yield (offset + data_offset, match.rule, name, value)
 
+    @staticmethod
```

Comment:
All three of these static methods seem to be the same across the two classes, shouldn't they just be part of the base class?